src/firefly/js/ui/Menu.jsx

```diff
@@ -218,18 +191,16 @@ function MenuTabBar({menuTabItems=[], size, selected, dropDown, displayMask, set
     const variant='soft';
     const color='primary';
 
-
     const tabItems= [
         <ResultsTab {...{key:'results-tab', size, color, variant, ref: (c) => setElement('results-tab ',c)}}/>,
         ...menuTabItems
-            .map(({action,label,title}, idx) =>
+            .map(({action,label,TabRenderer,title}, idx) =>
             {
-                const tab= (
-                    <Tab {...{ key: idx, value:action, disableIndicator:true, color, variant,
-                        ref: (c) => setElement(idx+'',c),
-                        sx: (theme) => ({ ...setupTabCss(theme,size) }) }} >
-                        {label}
-                    </Tab>);
+                const tabProps = {key: idx, value:action, disableIndicator:true, color, variant,
+                    ref: (c) => setElement(idx+'',c),
+                    sx: (theme) => ({ ...setupTabCss(theme,size) })
+                };
+                const tab= TabRenderer ? <TabRenderer {...tabProps} /> : <Tab {...tabProps}> {label}</Tab>;
```

Comment:
The ref is not being calls via `TabRenderer`. the setElement builds a object of all the tab widths for the  font size computation. The `Job Manager` tab via `TabRender` is never calling setElement so the size computation is wrong.

Reply:
My fault;  I correctly use `React.forwardRef` but forgot to set/forward the `ref`.  I'll fix it.
I'll also try to 'remove ALL showBgMonitor', there's so many.  As long as you say it's safe, I'll proceed.

Reply:
That was it.  It's fixed now.  Test URLs updated.